src/main/java/com/mixpanel/android/viewcrawler/ViewCrawler.java

```diff
@@ -812,8 +820,9 @@ private void updateEditState() {
             {
                 for (Pair<String, JSONObject> changeInfo:mEditorChanges.values()) {
                     try {
-                        final ViewVisitor visitor = mProtocol.readEdit(changeInfo.second);
-                        newEdits.add(new Pair<String, ViewVisitor>(changeInfo.first, visitor));
+                        final EditProtocol.Edit edit = mProtocol.readEdit(changeInfo.second);
+                        newEdits.add(new Pair<String, ViewVisitor>(changeInfo.first, edit.visitor));
```

Comment:
I agree! Referring to Visitors as "Edits" was meaningful before we had an "Edit" object that meant something different. I've changed `newEdits` to `newVisitors` in ba93c14